common/Cargo.toml

```diff
@@ -21,3 +21,4 @@ siphasher = "0.3"
 rand = "0.8"
 volatile = "0.3"
 radium = "0.6"
+libc = "0.2.101"
```

Comment:
The latest version of `libc` is `0.2.102` as of when I'm writing this comment. (See also #3050.)